scripts/generate/templates/post-purchase/src/index.tsx

```diff
@@ -72,78 +75,45 @@ export function App() {
     'Checkout::PostPurchase::Render'
   >();
   const initialState = storage.initialData as InitialState;
+
   return (
     <BlockStack>
```

Comment:
Can you increase the space between the Callout banner and the Layout? Loose will result in `21px` which we could probably use for the space between the columns in the main layout as well. This would look better.

```suggestion
    <BlockStack spacing="loose">
```

Our new Layout comes with a `spacing` prop which simplifies a lot things. Too bad we cant use it here.